### PR TITLE
Stop using deprecated methods from RobolectricHelpers

### LINKED ISSUE #4981: Stop using deprecated methods from RobolectricHelpers
#### Problem description
In `RobolectricHelpers` we have a few deprecated methods which we should stop using. Every method has its own description so it should be clear what we want to use instead.

### PR DESCRIPTION
**Pull request by dczerwinski**
Closes #4981 

#### What has been done to verify that this works as intended?
Gradle task "checkTests" is running successfully.

#### Why is this the best possible solution? Were any other approaches considered?
The methods I removed were deprecated and have been replaced by the methods mentioned in the comments.

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?
It should not affect users.

#### Do we need any specific form for testing your changes? If so, please attach one.
No.

#### Does this change require updates to documentation? If so, please file an issue [here]( https://github.com/getodk/docs/issues/new) and include the link below.
No.

#### Before submitting this PR, please make sure you have:
- [X] run `./gradlew checkAll` and confirmed all checks still pass OR confirm CircleCI build passes and run `./gradlew connectedDebugAndroidTest` locally.
- [X] verified that any code or assets from external sources are properly credited in comments and/or in the [about file](https://github.com/getodk/collect/blob/master/collect_app/src/main/assets/open_source_licenses.html).
- [X] verified that any new UI elements use theme colors. [UI Components Style guidelines](https://github.com/getodk/collect/blob/master/docs/CODE-GUIDELINES.md#ui-components-style-guidelines)

### CODE REVIEW

File: testshared/src/main/java/org/odk/collect/testshared/RobolectricHelpers.java [hunk 1]
```diff
@@ -99,38 +96,6 @@ public static void mountExternalStorage() {
         return null;
     }
```
Comment (grzesiek2010) [inline testshared/src/main/java/org/odk/collect/testshared/RobolectricHelpers.java]: I think it's a good opportunity to convert this class to kotlin.

Comment (dczerwinski) [reply testshared/src/main/java/org/odk/collect/testshared/RobolectricHelpers.java]: Converted.

File: testshared/src/main/java/org/odk/collect/testshared/RobolectricHelpers.kt [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+import org.robolectric.shadows.ShadowMediaPlayer
+import org.robolectric.shadows.ShadowMediaPlayer.MediaInfo
+import org.robolectric.shadows.util.DataSource
+
+object RobolectricHelpers {
+
+    var services: MutableMap<Class<*>, ServiceScenario<*>> = HashMap()
+
+    @JvmStatic
+    fun <T : FragmentActivity?> createThemedActivity(clazz: Class<T>?, theme: Int): T {
```
Comment (grzesiek2010) [inline testshared/src/main/java/org/odk/collect/testshared/RobolectricHelpers.kt]: This should not be nullable (both).

File: testshared/src/main/java/org/odk/collect/testshared/RobolectricHelpers.kt [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+        val activity = Robolectric.buildActivity(clazz)
+        activity.get()!!.setTheme(theme)
+        return activity.setup().get()
+    }
+
+    @JvmStatic
+    fun getCreatedFromResId(drawable: Drawable?): Int {
```
Comment (grzesiek2010) [inline testshared/src/main/java/org/odk/collect/testshared/RobolectricHelpers.kt]: `drawable` should not be nullable.

File: testshared/src/main/java/org/odk/collect/testshared/RobolectricHelpers.kt [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+        return Shadows.shadowOf(drawable).createdFromResId
+    }
+
+    @JvmStatic
+    @JvmOverloads
+    fun setupMediaPlayerDataSource(testFile: String?, duration: Int = 322450): DataSource {
```
Comment (grzesiek2010) [inline testshared/src/main/java/org/odk/collect/testshared/RobolectricHelpers.kt]: `testFile` should not be nullable.

File: testshared/src/main/java/org/odk/collect/testshared/RobolectricHelpers.kt [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+        ShadowMediaPlayer.addMediaInfo(dataSource, MediaInfo(duration, 0))
+        return dataSource
+    }
+
+    fun mountExternalStorage() {
+        ShadowEnvironment.setExternalStorageState(Environment.MEDIA_MOUNTED)
+    }
+
+    @JvmStatic
+    fun <T : ViewGroup?> populateRecyclerView(view: T): T {
```
Comment (grzesiek2010) [inline testshared/src/main/java/org/odk/collect/testshared/RobolectricHelpers.kt]: This should not be nullable.

**Review by grzesiek2010 — APPROVED** [review]

**Review by seadowg — APPROVED** [review]

**Pull request merged**